ext/ffi/00_ffi.js

```diff
@@ -142,6 +142,59 @@
     }
   }
 
+  function prepareParameters(args, types) {
+    const parameters = [];
+    const buffers = [];
+    let fn = null;
```

Comment:
I don't have the necessary understanding in Rust nor Deno's FFI codebase but I wonder if callbacks should be kept similarly as parameters and buffers in an array? This is of course my personal use case talking, since I know of an FFI function I really want to call from Deno that takes three callbacks as parameters...

Reply:
Very much a side note: I was wrong, I do not have a three-callback FFI function.